environment/visibility.csv

```diff
@@ -3,8 +3,8 @@ Mark,Sky Visibility,Start Date,End Date
 2406,good,2012-02-13T00:00:00Z,9999-01-01T00:00:00Z
 AHTI,dec. inc. 142 7 169 10,2009-01-01T00:00:00Z,9999-01-01T00:00:00Z
 AKTO,"Good to north, some fully grown trees to north east.",2006-04-27T00:00:00Z,9999-01-01T00:00:00Z
-ALS1,"Not great, trees to NE and SW, site on northern side of road.",2023-02-24T01:08:38Z,9999-01-01T00:00:00Z
-ALS2,Good,2023-02-23T01:17:02Z,9999-01-01T00:00:00Z
+ALS1,"Not great, trees to NE and SW, site on northern side of road.",2023-02-24T01:08:38Z,2024-02-25T13:59:30Z
+ALS2,Good,2023-02-23T01:17:02Z,2024-03-13T20:59:30Z
```

Comment:
@EvanCGNS Could you please remove hours, minutes and seconds from Start and End timestamps?